src/content/insertable-streams/video-processing/js/webgpu-background-blur.js

```diff
@@ -663,10 +697,59 @@ const batch = [4, 4];
 
     await device.queue.onSubmittedWorkDone();
 
+
+    const encoder = device.createCommandEncoder();
+    encoder.copyTextureToBuffer(
```

Comment:
would this copy only be required for workaround path?

Reply:
Right! Thanks for catching this!